dev/config/valid_emails.json

```diff
@@ -0,0 +1,3 @@
+[
+  "grid-demo-account@example.com"
+]
```

Comment:
I think that this file might have leaked in from auth work? I can't see anywhere it is referenced.